deployment/terraform/cluster.tf

```diff
@@ -108,6 +108,39 @@ resource "aws_rds_cluster" "db_cluster" {
   vpc_security_group_ids = ["${aws_security_group.db.id}"]
 }
 
+resource "aws_instance" "loadtest_agent" {
+  tags = {
+    Name = "${var.cluster_name}-agent-${count.index}"
+  }
+
+  connection {
+    type = "ssh"
+    user = "ubuntu"
+    host = self.public_ip
+  }
+
+  ami           = "ami-0fc20dd1da406780b"
+  instance_type = "t2.medium"
+  key_name      = aws_key_pair.key.id
+  count         = var.loadtest_agent_count
+
+  vpc_security_group_ids = ["${aws_security_group.agent.id}"]
+
+  provisioner "remote-exec" {
+      inline = [
+      "while [ ! -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting for cloud-init...'; sleep 1; done",
+      "wget https://dl.google.com/go/${var.go_binary_file}",
+      "sudo tar -C /usr/local -xzf ${var.go_binary_file}",
+      "wget https://github.com/mattermost/mattermost-load-test-ng/archive/${var.loadtest_source_code_ref}.tar.gz",
+      "tar -xzf ${var.loadtest_source_code_ref}.tar.gz",
+      "cp mattermost-load-test-ng-${var.loadtest_source_code_ref}/config/simplecontroller.default.json simplecontroller.json",
+      "cp mattermost-load-test-ng-${var.loadtest_source_code_ref}/config/config.default.json config.json",
+      "cp -r mattermost-load-test-ng-${var.loadtest_source_code_ref}/testdata testdata",
+      "cd mattermost-load-test-ng-${var.loadtest_source_code_ref} && export PATH=$PATH:/usr/local/go/bin && go build -o agent ./cmd/loadtest && sudo mv agent /usr/local/bin"
```

Comment:
Building binaries won't work if I am not wrong. That's what we have been trying to avoid. But we have all the hardcodings to profile images and testdata directories. So moving the binary to some place else will fail. Let's just do a "go run ./cmd/loadtest" and "go run ./cmd/coordinator".